src/auth/OAuthAuthenticator.js

```diff
@@ -28,6 +30,8 @@ var OAuthAuthenticator = function (options) {
 
   this.oauth = new RestClient(oauthUrl);
   this.clientId = options.clientId;
+  this.clientSecret = options.clientSecret;
+  this.domain = options.domain;
```

Comment:
where is the domain used? not familiar with the code base, but no change in the PR seems to be using it. if it is in `this.oauth` then 👍